nisync/session.py

```diff
@@ -864,6 +864,10 @@ def time_references(self):
     def resource_name(self):
         return self._resource_name
 
+    @property
+    def session_handle(self):
```

Comment:
Please add a simple regression test to ensure this doesn't change or go away in future changes.

Reply:
Added a regression test function named as "def test___session___compare_session_handle___succeeds(lib_mock)" in test_session.py